core/primitives/piece/piece_data.hpp

```diff
@@ -28,8 +29,12 @@ namespace fc::primitives::piece {
 
     bool isOpened() const;
 
+    bool getIsNullData() const;
+
    private:
     int fd_;
+    const bool isNullData;
```

Comment:
`NullReader` in lotus has size field (json too)

Reply:
In our case, it represents "/dev/zero". Infinite zero sequence. 

Size will be known when NullPiece will be written in sector file.